### PR TITLE
Bug 1550483 - move taskcluster-client.py in-tree

### PR DESCRIPTION
**Pull request by djmitche**
Bugzilla Bug: [1550483](https://bugzilla.mozilla.org/show_bug.cgi?id=1550483)

### CODE REVIEW

**Comment by djmitche** [comment]
@escapewindow ping for your review on this too, pending your accepting the invite to this repo.

The individual commits are well-isolated and make it easier to see what has changed in the generated code (very little).

**Comment by djmitche** [comment]
I suppose I'd like you to not get tripped up on something silly I've done next time you want to make a change -- could you make sure you can run the tests locally?

When this lands, I'll release a 7.1.0 to check that the release process works (it at least mechanically works, as I uploaded 7.0.1 to test.pypi.org -- https://test.pypi.org/project/taskcluster/)

**Comment by escapewindow** [comment]
`tox` works for me locally, thanks!

**Comment by djmitche** [comment]
It occurs to me that we can't use a pattern in .gitattributes here, since the `taskcluster/` and `taskcluster/aio/` directories both mix generated and non-generated code.  One option is to autogenerate `.gitattributes`, and another is to try to reorganize the code to better segregate generated and non-generated code.

**Comment by escapewindow** [comment]
Yeah, I wondered if we should make a `generated/` directory or something. It's possible we could craft imports in a way that this is seamless to users; also possible we create a one-time breaking change.

**Comment by djmitche** [comment]
We could probably leave compatibility "stubs" at the current location..

Anyway, this can be left for a followup.  Hopefully I'll get a taker from @taskcluster/services-reviewers to review this soon ;)

**Comment by djmitche** [comment]
Followup in [Bug 1551850](https://bugzilla.mozilla.org/show_bug.cgi?id=1551850)

File: clients/client-py/CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Community Participation Guidelines
```
Comment (imbstack) [inline clients/client-py/CODE_OF_CONDUCT.md]: We can probably remove this file, right?

File: clients/client-py/CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+# How to Contribute
```
Comment (imbstack) [inline clients/client-py/CONTRIBUTING.md]: And this one.

**Comment by djmitche** [comment]
Good point.  I think I'm just blind to those files since they're everywhere :)

**Pull request merged**